src/components/cdPipeline/cdpipeline.util.tsx

```diff
@@ -22,22 +22,31 @@ export const styles = {
             color: 'var(--N900)'
         })
     },
+    multiValue: (base ,state) => {
```

Comment:
The same style has been used on 2 places, please verify on both places

Reply:
done